addOns/bugtracker/src/main/java/org/zaproxy/zap/extension/bugtracker/BugTrackerGithubParam.java

```diff
@@ -38,7 +38,7 @@
     private static final String ALL_CONFIGS_KEY = GITHUB_BASE_KEY + ".configs.config";
 
     private static final String CONFIG_NAME_KEY = "name";
-    private static final String CONFIG_PASSWORD_KEY = "password";
+    private static final String CONFIG_TOKEN_KEY = "password";
```

Comment:
I left the config key named as "password", this isn't using versioned config and I wasn't sure it was worth changing.

Reply:
Better change.